docs/conf.py

```diff
@@ -16,32 +16,39 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('../'))
-print("ABSPATH: ", os.path.abspath('../'))
-
 from unittest.mock import MagicMock
 
+sys.path.insert(0, os.path.abspath("../"))
+
 
 class Mock(MagicMock):
     @classmethod
     def __getattr__(cls, name):
         return MagicMock()
 
 
-MOCK_MODULES = ['pyuvsim', 'aipy', 'aipy.const', 'pyuvdata', 'pyuvdata.utils',
-                'astropy', 'astropy.units', 'cached_property']
+MOCK_MODULES = [
+    "pyuvsim",
+    "aipy",
+    "aipy.const",
+    "pyuvdata",
+    "pyuvdata.utils",
+    "astropy",
+    "astropy.units",
+    "cached_property",
+]
 sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
 
 # -- Project information -----------------------------------------------------
 
-project = 'hera_sim'
-copyright = '2018, HERA-Team'
-author = 'HERA-Team'
+project = "hera_sim"
+copyright = "2020, HERA-Team"
+author = "HERA-Team"
 
 # The short X.Y version
-version = ''
+version = ""
```

Comment:
Do we want to update this to be 1.0?

Reply:
Actually, the version should be gotten automatically... I'll update this

Reply:
This should be done now.